### ISSUE TITLE
Xcode 13 / iOS 15 support

### ISSUE THREAD

**Issue by Drusy** [openium/SwiftiumTestingKit#6]
The following error occurs when testing using iOS 15
> -[UITouch setIsTap:]: unrecognized selector sent to instance 0x7fdfe4291be0

KIF update needed ?

**Comment by kenji21** [comment]
Right, reproduced within the TestApp:
```
2021-09-29 11:33:32.330756+0200 STKTestApp[85045:84425454] -[UITouch setIsTap:]: unrecognized selector sent to instance 0x128e43430
```

**Comment by kenji21** [comment]
Fixed in 1.1.0 : https://github.com/openium/SwiftiumTestingKit/releases/tag/1.1.0
